gammapy/time/lightcurve.py

```diff
@@ -320,6 +321,136 @@ def create_fixed_time_bin(time_step, spectrum_extraction):
             ])
         return intervals
 
+    def create_fixed_significance_bin_lc_table(self, significance, significance_method, energy_range,
+                                               spectrum_extraction, separators=[]):
+        """
+
+        Create time intervals such that each bin of a light curve reach a given significance and store information
+
+        Parameters
+        ----------
+        significance : float
+            Target significance for each light curve point
+        significance_method : {‘lima’, ‘simple’}
+            Significance method (see `~gammapy.stats.significance_on_off`)
+        energy_range : `~astropy.units.Quantity`
+            True energy range to evaluate integrated flux (true energy)
+        spectrum_extraction : `~gammapy.spectrum.SpectrumExtraction`
+            Contains statistics, IRF and event lists
+        separators : `list` of float
+            Contains a list of time (in MJD) to break the current point creation
+
+        Returns
+        -------
+        table : `~astropy.table.Table`
+            Table of time intervals  and information about their content : on/off events, alpha, significance
+
+        Examples
+        -------
+        extract intervals for light curve :
+            intervals = list(zip(table['t_start'], table['t_stop']))
+
+        """
+
+        def in_list(item, L):
+            o, j = np.where(L == item)
+            for index in o:
+                yield index
+
+        time_holder = []
```

Comment:
For better readability later, it might be useful to quickly document the structure of `time_holder`, e.g. stating which "type" (`start`, `end`, `break`, `on`, `off`) the second argument could be.

Reply:
good idea, I will this in the docstring